src/signer/src/test/java/ee/ria/xroad/signer/tokenmanager/merge/MergeOntoFileTokenStrategyTest.java

```diff
@@ -399,7 +401,7 @@ public void mergeKeyShouldOnlyCopyOverAvailabilityOutOfSingleFields() {
         final Token fileToken = new Token("fileToken", "fileId", CryptoUtils.CKM_RSA_PKCS_NAME);
         final Key fileKey = new Key(fileToken, fileId);
 
-        final String fileFriendlyName = "d§gsdasd";
+        final String fileFriendlyName = "d??gsdasd";
```

Comment:
Some characters have changed here. Is it intentional?

Reply:
These changes were unintentional.